pkg/api/controller.go

```diff
@@ -802,27 +810,76 @@ func (c *Controller) ListUsers(w http.ResponseWriter, r *http.Request, params Li
 	writeResponse(w, http.StatusOK, response)
 }
 
+func (c *Controller) generateResetPasswordToken(email string) (*string, error) {
+	secret := c.Auth.SecretStore().SharedSecret()
+	currentTime := time.Now()
+	token, err := GenerateJWTResetPassword(secret, email, currentTime, currentTime.Add(DefaultResetPasswordExpiration))
+	if err != nil {
+		return nil, err
+	}
+	return &token, nil
```

Comment:
Prefer the shorter version - I know it will modify the return type, but in the case of string it is common
```suggestion
	return GenerateJWTResetPassword(secret, email, currentTime, currentTime.Add(DefaultResetPasswordExpiration))
```

Reply:
Fixed